spring-integration-core/src/main/resources/org/springframework/integration/config/spring-integration-5.0.xsd

```diff
@@ -5009,6 +5009,18 @@ See 'SmartLifecycleRoleController'.
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
+	<xsd:attributeGroup name="maxFetchGroup">
+		<xsd:attribute name="max-fetch-size" type="xsd:string">
```

Comment:
`integer` ? :smile: 

Or something like property-placeholder or Bean Factory SpEL will bark, at least in the IDE?